docs/managing_deis/upgrading-deis.rst

```diff
@@ -63,6 +63,86 @@ Finally, update ``deisctl`` to the new version and reinstall:
     enable PROXY protocol for ports 80 and 443 on the ELB, add a ``TCP 443:443`` listener, and
     change existing targets and health checks from HTTP to TCP.
 
+
+Graceful Upgrade
+----------------
+
+Alternatively, an experimental feature exists to provide the ability to perform
+a graceful upgrade. This process is available for those upgrading from version
+1.9.0 or greater and is intended to facilitate upgrades within a major version.
+Upgrading between major versions is not supported. Unlike the in-place upgrade
+process described above, this process keeps the platform's routers and
+publishers up during most of the upgrade process to minimze application
+downtime.
+
+.. note::
+
+    The process utilizes a rolling restart of the platform's router components.
+    No more than one router instance is ever down at a time, but each stops and
+    restarts in fairly rapid succession.  Therefore, your load balancer
+    configuration is the determining factor in how much downtime, if any, will
+    occur during a successful upgrade.
+
+    Load balancers that are quick to remove a host from service after just one
+    or two failed health checks or are slow to return a host to service by
+    requiring many successful health checks to do so are the most likely to
+    incur small amounts of downtime.
+
+    You should review your load balancer's configuration to determine what to
+    expect during the upgrade process.
+
+The process involves two deisctl commands: ``upgrade-prep`` and
+``upgrade-takeover`` in coordination with a few other important commands.
+
+First, a new deisctl version (reflecting the desired version to upgrade
+to) should be installed to a temporary location. Care should be taken not to
+overwrite the existing deisctl version.
+
+.. code-block:: console
+
+    $ mkdir /tmp/upgrade
+    $ curl -sSL http://deis.io/deisctl/install.sh | sh -s 1.10.0 /tmp/upgrade
+    $ /tmp/upgrade/deisctl --version  # should match the desired version 1.10.0
```

Comment:
I thought it was potentially confusing to readers that the feature is for upgrading _from_ m >= 1.9.0 to n > m, but the docs showed an upgrade _to_ 1.8.0.

We could add a note that 1.10.0 does not exist yet.

This is why I was originally in favor of holding off on this documentation altogether until there was something to upgrade _to_ from 1.9.0.